projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.spec.ts

```diff
@@ -124,3 +148,44 @@ describe('ReportBPComponent', () => {
   });
 
 });
+
+describe('snackbar test', () => {
+  let fixture: ComponentFixture<ReportBPComponent>;
+  let loader: HarnessLoader;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [ReactiveFormsModule, NoopAnimationsModule, MatInputModule, MatCardModule],
+      declarations: [ReportBPComponent],
+      providers: [ MatSnackBar],
+    }).compileComponents();
+    fixture = TestBed.createComponent(ReportBPComponent);
+    fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+  });
+
+  it('should load harness for simple snack-bar', async () => {
+    const snackBarRef = fixture.componentInstance.open('Hi!', '');
+    let snackBars = await loader.getAllHarnesses(MatSnackBarHarness);
+
+    expect(snackBars.length).toBe(1);
```

Comment:
Please change this test's description to describe the expected behavior of the real code. We do not expect the app to load a test harness.

Reply:
Okay @semicolin